### ISSUE TITLE
Hard to parse

### ISSUE THREAD

**Issue by larseggert** [quicwg/base-drafts#4301]
>  As a result, the size of the packet number encoding is at least one
  bit more than the base-2 logarithm of the number of contiguous
  unacknowledged packet numbers, including the new packet.

That is really hard to parse.

>  For example, if an endpoint has received an acknowledgment for packet
  0xabe8bc, sending a packet with a number of 0xac5c02 requires a
  packet number encoding with 16 bits or more; whereas the 24-bit
  packet number encoding is needed to send a packet with a number of
  0xace8fe.

and so is that.

Indeed I found the remaining paras of the section difficult to follow.

**Comment by ianswett** [comment]
I agree that first sentence is hard to parse.

**Comment by martinthomson** [comment]
I also agree regarding the first.  As this is an alternative description, and I don't have anything better to offer, I plan not to change anything unless someone offers something good.

The second seems fine.

**Comment by MikeBishop** [comment]
I wonder if the real takeaway here is that these "examples in passing" assume the reader is deeply following along with the algorithm in their head; I know I don't, but I'm also not implementing packetization.  We already have an appendix that illustrates packet number decoding; perhaps we should move one or both paragraphs to an appendix that illustrates selecting an encoding, with these as the examples.

Reading the rest of the section, is "protection of the packet number" the term we've used elsewhere?  Shouldn't that be "header protection," perhaps with a cross-ref to -tls?
